code/style.css

```diff
@@ -1,58 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background-color: #fdcb9e;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.title {
+  text-align: center;
 }
 
-p {
+.deck-title {
+  font-weight: bold;
+  background-color: #fff;
+}
+
+.deck-content {
   margin: 1px;
   font-weight: 300;
   background-color: #fff;
 }
 
+h1 {
+  text-align: center;
+}
+
 .cards {
   margin: 50px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
-  justify-content: space-around;
+  flex-direction: column;
   align-items: center;
 }
 
-.killer-deck {
+.deck {
   width: 250px;
-  height: 400px;
-  background: #000;
+  height: 440px;
   text-align: center;
   padding-top: 30px;
+  margin-bottom: 16px;
   border-radius: 16px;
+  box-shadow: 10px 10px 5px 0px rgba(0, 0, 0, 0.3);
+}
+
+.killer-deck {
+  background: #1b262c;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  background: #0f4c75;
 }
 
 .room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+  background: #00b7c2;
+}
 
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+.deck-heartbeat {
+  animation: 0.8s 2 heartbeat;
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: #ec4d73;
   border: none;
   font-size: 20px;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: 20px auto;
+  box-shadow: 6px 6px 3px 0px rgba(0, 0, 0, 0.3);
+}
+
+@keyframes heartbeat {
+  0% {
+    transform: scale(1);
+  }
+  20% {
+    transform: scale(1.1);
+  }
+  40% {
+    transform: scale(1);
+  }
+  60% {
+    transform: scale(1.1);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+
+/***** MEDIA QUERIES - TABLET and DESKTOP ****/
+@media (min-width: 481px) {
```

Comment:
This is so clean - having to add just a few lines to make the site responsive. Yay!